src/PaymentGateways/Gateways/Stripe/resources/js/StripeAccountPanel.tsx

```diff
@@ -0,0 +1,108 @@
+import {__} from '@wordpress/i18n';
+import {InspectorControls} from '@wordpress/block-editor';
+import {Notice, PanelBody, SelectControl, ToggleControl} from '@wordpress/components';
+import {createInterpolateElement} from '@wordpress/element';
+import {getFormBuilderWindowData} from '@givewp/form-builder/common/getWindowData';
+import './styles.scss';
+
+export default function StripeAccountPanel({
+    attributes,
+    setAttributes,
+}: {
+    attributes: StripeProps;
+    setAttributes: ({}) => void;
+}) {
+    const {gatewaysGlobalSettings, gatewaySettingsUrl} = getFormBuilderWindowData();
+    const stripeGlobalSettings: StripeGlobalSettingsProps = gatewaysGlobalSettings?.stripe_payment_element ?? {};
+
+    const textWithLinkToStripeSettings = (textTemplate: string) =>
+        createInterpolateElement(textTemplate, {
+            a: <a href={`${gatewaySettingsUrl}&section=stripe-settings&group=accounts`} />,
```

Comment:
This whole url looks like it should come from the window instead of building it here.

Reply:
Sure

Reply:
Resolved: https://github.com/impress-org/givewp/pull/6903/commits/c5a70ef35e2b5f8d1de15e0061bb2d392ab48bf2